db/migrate/20240226193757_migrate_templates_to_mappings.rb

```diff
@@ -0,0 +1,8 @@
+class MigrateTemplatesToMappings < ActiveRecord::Migration[7.0]
+  def up
+    MappingMigrationService.new.call
+  end
+
+  def down
```

Comment:
Not important, but do we need to add a reverse of the migration service? It would

- Delete the Mapping and MappingFields
- Rename the .legacy template files to .template